tests/system/action/meeting/test_import.py

```diff
@@ -363,7 +363,6 @@ def get_user_data(self, obj_id: int, data: Dict[str, Any] = {}) -> Dict[str, Any
             "is_demo_user": False,
             "organization_management_level": None,
             "is_present_in_meeting_ids": [],
-            "submitted_motion_$_ids": [],
```

Comment:
Please move the removed templates field to an instance of meeting_user for import, testing also the import

Reply:
There is a test for "personal_note_ids". I won't improve the tests here.